<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SavingChanges</class>
 <widget class="QDialog" name="SavingChanges">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сохранить изменения?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="butt_yes">
         <property name="text">
          <string>Да</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butt_no">
         <property name="text">
          <string>Нет</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butt_cancel">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
